libc/stdlib/itoa.c

```diff
@@ -0,0 +1,34 @@
+#include <string.h>
+#include <stdlib.h>
+#include <stdbool.h>
+
+
+char get_ascii(int number){
```

Comment:
I don't understand how it works, if it is number from 0-9 we convert it into char equivalent. and number 11 means A right? What about small letters there is a gap in ASCI between big and small case letters. 
Please elaborate
![image](https://user-images.githubusercontent.com/58564314/199630227-765e6a3e-7923-4201-9d43-fba00fd118c8.png)